apps/frontend/src/modules/auth/slices/actions.ts

```diff
@@ -2,14 +2,37 @@ import { createAsyncThunk } from "@reduxjs/toolkit";
 
 import { StorageKey } from "~/libs/modules/storage/storage.js";
 import { type AsyncThunkConfig } from "~/libs/types/types.js";
+import { type AuthResponseDto } from "~/modules/auth/auth.js";
+import { NotificationMessage } from "~/modules/auth/libs/enums/enums.js";
 import {
 	type UserResponseDto,
+	type UserSignInRequestDto,
 	type UserSignUpRequestDto,
 	type UserSignUpResponseDto,
 } from "~/modules/users/users.js";
 
 import { name as sliceName } from "./auth.slice.js";
 
+const signIn = createAsyncThunk<
+	AuthResponseDto,
+	UserSignInRequestDto,
+	AsyncThunkConfig
+>(`${sliceName}/sign-in`, async (signInPayload, { extra, rejectWithValue }) => {
+	const { authApi, notification } = extra;
+
+	try {
+		const response = await authApi.signIn(signInPayload);
```

Comment:
```suggestion
		const { user, token } = await authApi.signIn(signInPayload);
```
And return only user